### DISCUSSION TITLE
What is the HS version?

### DISCUSSION THREAD

**Discussion by dn32** [Jyers/Marlin#1238] [Q&A] [answered]
I would like to know what is different between these two versions. I looked in the documentation, but I couldn't find it. If you have a documentation link to refer me, I'm very grateful.

> E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v1.3.5
> E3V2-UBL-BLTouch-10x10-v4.2.2-v1.3.5

**Comment by Tuncay-Ayhan** [comment, marked as answer]
> I would like to know what is different between these two versions. I looked in the documentation, but I couldn't find it. If you have a documentation link to refer me, I'm very grateful.
> 
> > E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v1.3.5
> > E3V2-UBL-BLTouch-10x10-v4.2.2-v1.3.5

HS stands for High Speed. 

The regular version retracts the probe when moving. The HS version takes less time with probing, because the probe doesn't have to retract itself while moving to a new probe location.
